src/operun/crm/browser/account.py

```diff
@@ -1,5 +1,6 @@
 # -*- coding: utf-8 -*-
 from plone import api
+from Products.CMFCore.utils import getToolByName
```

Comment:
Please use the api method to get a tool. See https://docs.plone.org/develop/plone.api/docs/api/portal.html#plone.api.portal.get_tool